hermeznetwork/bridge-ui
# Add the 1Inch price oracle

### What does this PR does?

This PR adds a context to query the 1Inch price oracle for a token price.

### How to test?

To try the USD conversion, you can write this piece of code in the `home.view.tsx` file. It should return the price in USD of the `MATIC` token.

```ts
useEffect(() => {
  getTokenPrice("0x7D1AfA7B718fb893dB30A3aBc0Cfc608AaCfeBB0")
    .then((tokenPrice) => {
      console.log(formatUnits(tokenPrice, USDT_DECIMALS));
    })
    .catch((err) => console.log(err));
}, [getTokenPrice]);
```
## Checklist

These are the criteria that every PR should meet, please check them off as you
review them:
